packages/node_modules/@webex/webex-core/test/unit/spec/credentials/credentials.js

```diff
@@ -228,6 +228,22 @@ describe('webex-core', () => {
       });
     });
 
+    describe('#extractOrgIdFromUserToken()', () => {
+      let credentials;
+      let webex;
+
+      beforeEach('generate webex and destructure credentials', () => {
+        webex = new MockWebex();
+        credentials = new Credentials(undefined, {parent: webex});
+      });
+
+      it('should throw when provided an invalid token', () =>
```

Comment:
I think the "good" case is missing, when the orgID is actually returned

Reply:
I did not implement a **good** case for the unit test, but it exists in the integration test [as it requires a valid user token].

Reply:
Thanks for adding it!
The value and I see in having a unit test, in addition to the integration is that it allows for developers that didn't write the code to see at glance what expected return values look like. For instance, now I know that the org ID returned is an base64 org ID value and not a name or other type of ID